tests/kafkatest/tests/core/compatibility_test_new_broker_test.py

```diff
@@ -46,12 +45,12 @@ def setUp(self):
 
     @cluster(num_nodes=6)
     @parametrize(producer_version=str(LATEST_0_8_2), consumer_version=str(LATEST_0_8_2), compression_types=["none"], new_consumer=False, timestamp_type=None)
-    @parametrize(producer_version=str(LATEST_0_8_2), consumer_version=str(LATEST_0_9), compression_types=["none"], new_consumer=False, timestamp_type=None)
```

Comment:
It It seems a bit odd that we have so many combinations for 0.8.2 and 0.9 while we have very few for each of the 0.10.x versions. Note that we quite a few cases where the consumer version is LATEST_0_9.